.gitignore

```diff
@@ -1,2 +1,4 @@
 *.jl.*.cov
 *.jl.mem
+deps/ext.jl
```

Comment:
Where do these come from? We haven't had a `deps` folder in a loong time.

Reply:
Dunno, but I dropped it.